python/ray/serve/_private/constants.py

```diff
@@ -154,7 +154,7 @@
 RECOVERING_LONG_POLL_BROADCAST_TIMEOUT_S = 10.0
 
 # Minimum duration to wait until broadcasting model IDs.
-PUSH_MULTIPLEXED_MODEL_IDS_INTERVAL_S = 1.0
+PUSH_MULTIPLEXED_MODEL_IDS_INTERVAL_S = 0.1
```

Comment:
btw, this is going to cause a lot of additional traffic in steady state. is there a simple way to push the model_ids only on update and avoid broadcasting so frequently all the time?

Reply:
it won't increase unnecessary traffics. we have a guard https://github.com/ray-project/ray/blob/master/python/ray/serve/multiplex.py#L138.
This is to push the information as early as possible.